src/components/App.js

```diff
@@ -1,20 +1,113 @@
-import React from 'react'
+import { useDataQuery } from '@dhis2/app-runtime'
+import PropTypes from 'prop-types'
+import React, { useState, useEffect } from 'react'
 import { connect } from 'react-redux'
+import { acClearCurrent, acSetCurrent } from '../actions/current'
+import {
+    acClearVisualization,
+    acSetVisualization,
+} from '../actions/visualization'
+import history from '../modules/history'
+import { sGetCurrent } from '../reducers/current'
+import { sGetVisualization } from '../reducers/visualization'
 import classes from './App.module.css'
-import VisualizationOptionsManager from './VisualizationOptions/VisualizationOptionsManager'
+import { Toolbar } from './Toolbar/Toolbar'
 
-const App = () => (
-    <>
+const visualizationQuery = {
+    eventReport: {
+        resource: 'eventReports',
+        id: ({ id }) => id,
+        // TODO copy pasted from old ER
```

Comment:
Yes, check if we want/need the exact same list or not.
This list is slightly different, it has `href` enabled because `FileMenu` needs it for the TranslationDialog to work without passing a d2 Model.

Reply:
Fixed the TODO comment